examples/website-demos/social-messaging/src/components/Thread/SocialThread.scss

```diff
@@ -1,67 +1,67 @@
-@import './_variables';
+@import '../../styles/_variables';
 
-.str-chat {
-  &__li {
-    margin-top: 0;
-    margin-bottom: 0;
-  }
-  &__thread {
-    max-width: unset;
-
-    &-list {
-      padding: unset;
+.thread {
+  &-header {
+    &-container {
+      display: flex;
+      background: $app-canvas;
+      transition: background 1s;
+      align-items: center;
+      justify-content: flex-start;
+      width: 100%;
+      padding: 8px;
+      border-bottom: 1px solid $stroke;
+      height: 60px;
+      box-sizing: border-box;
     }
 
-    .thread-header-container {
-      border: none;
-      border-bottom: 1px solid $stroke;
+    &-close {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      padding: 16px;
+      cursor: pointer;
+
+      svg {
+        fill: $black;
+
+        &:hover path {
+          fill: $accent-primary;
+        }
+      }
     }
 
-    &-start {
-      display: none;
+    &-title {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      justify-content: center;
+      color: $text-high-emphasis;
+      font-weight: 500;
+      font-size: 17px;
+      padding-left: 16px;
+
+      &-sub-title {
+        color: $text-low-emphasis;
+        font-size: 13px;
+        font-weight: 400px;
+      }
     }
   }
 }
 
-.message-ui {
-  margin-top: auto;
-}
-
-.str-chat__thread .str-chat__message-replies-count-button {
-  display: none;
+.str-chat__thread {
+  margin: 0 !important;
+  border-left: 1px solid $stroke !important;
+  flex: 1 !important;
+  max-width: none !important;
```

Comment:
nest this selector to try and get rid of the importants, this comment goes for all importants in the PR

Reply:
on it